File: eRestoran/eRestoran.Model/Recenzija.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eRestoran.Model
{
    public class Recenzija
    {
        public int Id { get; set; }
        public int Ocjena { get; set; }
        public string Opis { get; set; }
        public int JeloId { get; set; }
        //public int KorisnikId { get; set; }
    }
}

File: eRestoran.Mobile/lib/models/Recenzija.dart
import 'dart:convert';

class Recenzija {
     final  int Ocjena; 
     final String  Opis;
     final int JeloId ;
     final int KorisnikId;

  Recenzija({required this.Ocjena, 
           required this.Opis, 
           required this.JeloId,
           required this.KorisnikId});


  factory Recenzija.fromJson(Map<String, dynamic> json) {
    return Recenzija( 
        Ocjena: json["Ocjena"],
        Opis: json['Opis'],
        JeloId: json['JeloId'],
        KorisnikId: json["KorisnikId"]
    );
  }

  Map<String, dynamic> toJson() => {
        "Ocjena": Ocjena,
        "Opis": Opis,
        "JeloId": JeloId,
        "KorisnikId":KorisnikId
      };
}
